Tests/SentryTests/SentryClientTests.swift

```diff
@@ -58,8 +58,22 @@ class SentryClientTest: XCTestCase {
         super.setUp()
         fixture = Fixture()
     }
-
+    
     func testCaptureMessage() {
+        let eventId = fixture.getSut().capture(message: message, scope: nil)
```

Comment:
L: if you don't need a `scope`, I believe it makes sense to have an overload of eg `captureEvent(Event)`, we do on Java/Android

Reply:
I agree, but I don't want to do this in this PR. I added a task in Asana for this.